.github/workflows/update_github_io.yml

```diff
@@ -15,18 +15,33 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the latest code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+          token: ${{ secrets.PAT }}
+      - name: Make Temp Folder
+        run: |
+          mkdir .ci-temp
+          cd .ci-temp
+      - name: Checkout io repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.PAT }}
+          repository: checkstyle/checkstyle.github.io
+          path: checkstyle.github.io
       - name: Run Shell Script
         run: |
           ./.ci/update-github-io.sh ${{ github.event.inputs.version }}
-      - name: Commit and Push
+      - name: Commit
         run: |
-          git config --global user.name 'github-actions[bot]'
-          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
-          cd . ci-temp/checkstyle.github.io
+          cd checkstyle.github.io
+          git rm -rf *
```

Comment:
We are really removing all files?

Reply:
I have taken the reference from `release.sh` -> https://github.com/checkstyle/checkstyle/blob/master/release.sh#L131

Reply:
Yes, remove all, to have no history in repo, to make repo quick to checkout

Reply:
I do not see copy of new files to repo
https://github.com/checkstyle/checkstyle/blob/7658776d784d82531db387027d5bfb1cb7cde0a9/release.sh#L133

Reply:
https://github.com/Rahulkhinchi03/checkstyle/runs/8253537139?check_suite_focus=true#step:5:14 didn't match any file